packages/api/lib/assertions.js

```diff
@@ -13,3 +13,12 @@ exports.isUnauthorizedUserResponse = (t, response) => {
   const responseBody = JSON.parse(response.body);
   t.is(responseBody.message, 'User not authorized');
 };
+
+// The error message when change once handle() changes from using
+// resp() to getAuthorizationFailureResponse()
+exports.isInvalidTokenResponse = (t, response) => {
+  t.is(response.statusCode, 400);
+
+  const responseBody = JSON.parse(response.body);
+  t.is(responseBody.message, '"Invalid Authorization token"');
```

Comment:
Why is the message in quotes?

Reply:
For some reason the message was coming back that way and it looks like I forgot to double-back as to why. It's odd because the rest of the messages are not coming back in quotes, as you can see. I'll take another look